app/views/admin/consent_limits/index.haml

```diff
@@ -0,0 +1,29 @@
+.row
```

Comment:
Add breadcrumbs for consistency?

Reply:
Yes, but on edit and new.  Added tabs here.  Thanks.